pkg/cleaner/aws/aws.go

```diff
@@ -62,7 +87,33 @@ func (a *Cleaner) Clean() error {
 	return nil
 }
 
-func shouldBeDeleted(stack *cloudformation.Stack) bool {
+func (a *Cleaner) cleanBuckets() error {
+	input := &s3.ListBucketsInput{}
+	output, err := a.s3Client.ListBuckets(input)
+	if err != nil {
+		return err
+	}
+	for _, bucket := range output.Buckets {
+		a.logger.Log("level", "debug", "message", fmt.Sprintf("checking bucket %q", *bucket.Name))
```

Comment:
The last idea in `fmt` was `%#q`, so it is quoted in backticks.

Reply:
Done.